Team_Roster/pages/stats.py

```diff
@@ -0,0 +1,317 @@
+from app import db
```

Comment:
What's going on here? This significantly changes how everything is arranged in the app, so I think it's a bit late for this - let's just add the `getPitchingStats` to the roster.py so it is consistent with everything else.